config/jobs/kubernetes/sig-scalability/sig-scalability-presets.yaml

```diff
@@ -38,11 +38,6 @@ presets:
   # Reduce etcd compaction frequency to match production.
   - name: KUBEMARK_ETCD_COMPACTION_INTERVAL_SEC
     value: "150"
-  # Use Taint based evictions to control hollow node recreation in case of
-  # node VM restart.
-  # See https://github.com/kubernetes/kubernetes/issues/67120 for context.
-  - name: KUBE_FEATURE_GATES
-    value: "TaintBasedEvictions=true"
```

Comment:
TaintBasedEviction was enabled by default already in 1.16:
https://github.com/kubernetes/kubernetes/blob/release-1.16/pkg/features/kube_features.go#L520

So you can just safely remove it from everywhere - it was a noop for quite some time...

Reply:
ah cool, thanks